iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/ProvisioningTests.java

```diff
@@ -119,6 +119,21 @@ public void individualEnrollmentGetAttestationMechanismTPM() throws Provisioning
         assertNotNull(retrievedTpmAttestation.getEndorsementKey());
     }
 
+    @Test
+    public void recreateProviderWithKeyTPM() throws Exception
```

Comment:
Does Java follow camel casing rules, including acronyms?
```suggestion
    public void recreateProviderWithKeyTpm() throws Exception
```